src/main/java/io/anserini/search/RetrieveMain.java

```diff
@@ -0,0 +1,71 @@
+package io.anserini.search;
```

Comment:
Let's rename this `SearchMsmarco`? Looks a bit ugly that's what camel case convention would dictate?

Let's add a binding to generate a launch script, per https://github.com/castorini/anserini/blob/master/pom.xml#L106